app/src/layouts/appLayout/appSidebar/appSidebar.jsx

```diff
@@ -77,7 +77,9 @@ export class AppSidebar extends Component {
     extensionComponents: [],
     onClickNavBtn: () => {},
   };
-
+  state = {
+    isProjectsListOpen: false,
```

Comment:
Do we really need to hold an additional flag for `project list` visibility?
I think the tooltip availability can be controlled inside of the `ProjectSelector` component by already existing `opened` state property.
You just need to extract `withTooltip` decorator from `ProjectSelector` itself and wrap the necessary stuff inside of `ProjectSelector` with it.